news/2024/2024-07-12-osu-catch-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,394 @@
+---
+layout: post
+title: osu!catch World Cup 2024 Concludes
+date: 2024-07-12 23:00:00 +0000
+---
+
+After years of South Korean domination, this year we finally crown a new osu!catch World Cup champion.
+
+![](/wiki/Tournaments/CWC/2024/img/cwc2024-banner.jpg)
+
+The United States, persevering after many podium finishes, manage for the first time in their history to win the osu!catch World Cup. They were the favourites coming into this tournament, and tore through their bracket to reach the Grand Finals with only a single map lost, and in the end, no one could stop them. It's only the fourth different country to win this tournament, after Chile, South Korea, and China, so it is a big day for our game mode.
+
+## Summary
+
+![](/wiki/Tournaments/CWC/2024/img/cwc2024-podium.jpg)
+
+Congratulations to the United States for their historic first osu!catch World Cup victory, and congratulations to Argentina and Poland on their podium finishes.
+
+The United States have been the "challengers" for years at this point, but were never the favourites to win it all against China and South Korea. They were however, always a top 6 contender and more often even 2nd and 3rd. With a lot of their core members remaining the same over the years, it is truly a special moment for them to finally become champions, after so many closes finishes.
+
+We also have an (almost) entirely new podium this year, with Poland making their first appearance up there, along with Argentina who hadn't made top 3 since 2013. Both of these teams have had their best run ever this tournament, and thanks to their hard work and dedication, they finally reached the p
```

Comment:
```suggestion
We also have an (almost) entirely new podium this year, with Poland making their first appearance up there, along with Argentina which hasn't made top 3 since 2013. Both of these teams have had their best run ever this tournament, and thanks to their hard work and dedication, they finally reached a new peak.
```